### ISSUE TITLE
OSError: [Errno 22] Invalid argument when try browser_cookie3.chrome

### ISSUE THREAD

**Issue by skyfire000** [borisbabic/browser_cookie3#43]
when i tried to load cookie from the Chromium Edge , it have the error

```
browser_cookie3.chrome(cookie_file='C:\\Users\\JX\\AppData\\Local\\Microsoft\\Edge\\User Data\\Default\\Cookies')

Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "G:\Wpy\WPy64-3810\python-3.8.1.amd64\lib\site-packages\browser_cookie3\__init__.py", line 351, in chrome
    return Chrome(cookie_file, domain_name).load()
  File "G:\Wpy\WPy64-3810\python-3.8.1.amd64\lib\site-packages\browser_cookie3\__init__.py", line 177, in load
    expires = expires.timestamp()
OSError: [Errno 22] Invalid argument
```

My Env is  Winpython64,python3.8.1 at Win10 x64

but it works when I tried 
```
browser_cookie3.chrome(cookie_file='C:\\Users\\JX\\AppData\\Local\\Microsoft\\Edge\\User Data\\Default\\Cookies', domain_name="github.com")

```
and it works with browser_cookie3.firefox()

**Comment by brooksmcmillin** [comment]
This looks like the same issue as https://github.com/borisbabic/browser_cookie3/issues/39. I have a quick patch in the linked thread and my recent pull request should fix this as well.

**Comment by borisbabic** [comment]
This should be fixed in >=0.10.1
